app/uk/gov/nationalarchives/omega/editorial/services/jms/ResponseBuilder.scala

```diff
@@ -22,27 +22,31 @@
 package uk.gov.nationalarchives.omega.editorial.services.jms
 
 import cats.MonadError
+import cats.effect.IO
 import cats.implicits._
 import jms4s.jms.JmsMessage
-import org.typelevel.log4cats.Logger
 import play.api.libs.json.{ Json, Reads, Writes }
 import uk.gov.nationalarchives.omega.editorial.connectors.ApiConnector.SID
 import uk.gov.nationalarchives.omega.editorial.models._
-import uk.gov.nationalarchives.omega.editorial.services.jms.ResponseBuilder.ME
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+import org.typelevel.log4cats.SelfAwareStructuredLogger
 
-class ResponseBuilder[F[_] : ME : Logger] extends StubData {
+import javax.inject.{ Inject, Singleton }
+
```

Comment:
Why the change from `F[_]` to `IO[_]`?

Reply:
In short - we were not using it, and I need to be able to inject this class with Guice. Type parameters make that a lot harder.

We're not using any other effect type here instead of IO. If we were using generic effects then I would consider keeping it, as it is, I think it's unnecessarily complicated and doesn't fit the style of the rest of this application (using Guice)

Reply:
That's what I thought: we don't need the flexibility.

I think it's a good step.